toolkit.lisp

```diff
@@ -167,3 +167,11 @@
 
 (defun coerce-boolean (boolean provided)
   (when provided (if boolean "true" "false")))
+
+(defgeneric maybe-parse-integer (object))
```

Comment:
`maybe-parse-integer` is somewhat confusing of a name. I'd call it `ensure-integer` instead.